guided-meditations.md

```diff
@@ -436,6 +442,12 @@ Start
         Download "Hello, {name}! Does this meet your expectations"
           To "example.txt" MimeType "text/plain"
         Stop
+    When "I have so many things"
+      Form
+        things = Paste /[A-Z]\d{3,}/ With Label "Can you list all the things?"
+      Then
+        RepeatFor thing In things: Table "A Thing" "Try not to let {thing} bother you."
```

Comment:
Thanks @apmasell for this! I tried out the example, it looks really useful.

Small fix, I had to modify `Table "A Thing"` to `Table Column "A Thing"`.

Reply:
Oops. Will do.